app/controllers/collections_controller.rb

```diff
@@ -49,6 +49,10 @@ def update
     end
   end
 
+  def show
+    @collection = Collection.find(params[:id])
```

Comment:
This appears to be untested.

Reply:
I added a request spec, this should be covered now.